servlet/src/test/java/io/undertow/servlet/test/dispatcher/util/DispatcherUtil.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
```

Comment:
Please, update the year to 20223.

Reply:
@fl4via I've updated the changes to 2023.